app/src/main/java/com/example/marvelapp/mapper/MarvelMapper.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.marvelapp.mapper
+
+import com.example.marvelapp.entity.Character
+import com.example.marvelapp.service.response.DataResponse
+import org.koin.core.component.KoinComponent
+
+class MarvelMapper : KoinComponent{
```

Comment:
KoinComponent is not necessary as you are not injecting anything. In fact, you can remove this and use an extension function

Reply:
fixed